app/components/elements/Userpic.jsx

```diff
@@ -20,7 +20,7 @@ class Userpic extends Component {
         } catch (e) {}
 
         if (url && /^(https?:)\/\//.test(url)) {
-            const size = width && width > 48 ? '320x320' : '72x72'
+            const size = width && width > 48 ? '320x320' : '120x120';
```

Comment:
96x96 works good for the retina 13/15", but on 4k still comes soft since it's around 108.79 PPI/27" @4x.  At the current dimensions it has full sharpness on a 32" 4k.